benchmarks/dbscan/ArborX_DBSCANVerification.hpp

```diff
@@ -18,6 +18,7 @@
 
 #include <Kokkos_Core.hpp>
 
+#include <iostream>
```

Comment:
Do we also want to include `<cstdio>` here given that the code uses `printf`?

Reply:
For this particular file I would say no because all printf call are on the device side and we eventually will probably just use Kokkos::printf